wavefront/server/apps/floware/floware/services/scheduled_job_service.py

```
@@ -0,0 +1,417 @@
+        )
+        if not acquired:
+            # Another worker already executed this fire time. Release the row-level
+            # lock that claim_due_jobs placed on the row so the job is not stuck
+            # in 'running' until stale-lock recovery fires (30 min later).
+            # We also advance next_run_at so the same execution_key is never hit
+            # again, breaking the otherwise-infinite claim → duplicate → recover loop.
+            try:
+                next_run_at = self._compute_next_run_at(
+                    job_row['cron_expr'], job_row['timezone']
+                )
+            except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)